lib/ModuleError.js

```diff
@@ -2,15 +2,20 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-function ModuleError(module, err) {
-	Error.call(this);
-	Error.captureStackTrace(this, ModuleError);
-	this.name = "ModuleError";
-	this.module = module;
-	this.message = err;
-	this.error = err;
+"use strict";
+
+class ModuleError extends Error {
+
+	constructor(module, err) {
+		super();
+		if(Error.hasOwnProperty("captureStackTrace")) {
+			Error.captureStackTrace(this, this.constructor);
```

Comment:
This should be `new.target` instead of `this.constructor`... but Node 4 doesn't support it :cry: